transformer_engine/pytorch/transformer.py

```diff
@@ -343,8 +590,8 @@ def __init__(
                 **common_gemm_kwargs,
             )
 
-        # Core Self attention.
-        self.core_attention = CoreAttention(
+        # Attention.
+        self.dot_product_attention = DotProductAttention(
```

Comment:
```suggestion
        self.core_attention = DotProductAttention(
```

From the point of view of the multiheadattention module that is its "core" attention mechanism :-).